src/function-configuration.ts

```diff
@@ -5,11 +5,15 @@ export const SUPPORTED_REGIONS = [
   'us-central1',
   'us-east1',
   'us-east4',
+  'us-west3',
```

Comment:
Now that GCF supports so many region could we consider changing the invalid region check from an error to a warning?  That way we don't have to keep chasing this list.

We can link people to this page from the warning:
https://cloud.google.com/functions/docs/locations

Reply:
I'm not actually going to have it error or warn anymore. The supported regions will just be a TypeScript hinting convenience, and in a followup I think we should have the CLI do a live check against the locations API instead.